src/routers/userRoute.ts

```diff
@@ -45,5 +46,6 @@ router.patch(
 );
 
 router.get('/users/verify-email/:token', verifyEmail);
+router.post('/users/:id/accountStatus', isAuthenticated, isAdmin, userStatus);
```

Comment:
we have naming conversation of api like if verifyEmail should be verify-email or account-status